bin/cheritest/cheritest_registers.c

```diff
@@ -474,6 +474,33 @@ test_initregs_stack(const struct cheri_test *ctp __unused)
 		cheritest_failure_errx("tag %jx (expected 1)", v);
 	cheritest_success();
 }
+
+void
+test_initregs_returncap(const struct cheri_test *ctp __unused)
+{
+	/* The return capability should always be a sentry capability */
+	void *retcap = __builtin_return_address(0);
+	uintmax_t perms = cheri_getperm(retcap);
```

Comment:
```suggestion
	void *retcap;
	uintmax_t v;
	
	/* The return capability should always be a sentry capability */
	c = __builtin_return_address(0);
	v = cheri_getperm(retcap);
```
(and update perms to v everywhere)

Reply:
Not keen separate local variable declaration+initialization (and having them all declared at the start). I also don't really like reusing generic variables names such as `v` for different things.

But I guess being consistent is more important than personal preference.

Reply:
Yeah. I care less about the `v` as that's somewhat dubious, but we should adhere to style(9).